app/views/template_invocations/show.html.erb

```diff
@@ -18,7 +18,8 @@ end
 
 <div id="title_action">
   <div class="btn-toolbar pull-right">
-    <%= link_to(_('Back to Job'), job_invocation_path(@template_invocation.job_invocation), :class => 'btn btn-default') %>
+    <%= button_group(link_to(_('Back to Job'), job_invocation_path(@template_invocation.job_invocation), :class => 'btn btn-default'),
+                     link_to(_('Rerun'), rerun_job_invocation_path(@template_invocation.job_invocation, :host_ids => [ @host.id ]), :class => 'btn btn-default', :permission => :create_job_invocations)) %>
```

Comment:
Not sure what were the expectations about including `:permission => :create_job_invocations` but it doesn't seem to do anything. From what I'm seeing it just sets `permission="create_job_invocations"` attribute on the link tag.

Reply:
My bad, now it should be correct :+1: